### PR TITLE
doc(release): update release instructions

### PR DESCRIPTION
**Pull request by KhaledEmaraDev**
## Checklist

- [] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [] I have inspected the website preview for accuracy.
- [] I have signed off my issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,17 +120,38 @@ To create a new release branch:
...
 
-2. Update `version_menu` and `version` in [params.toml](/config/_default/params.toml) for the next release.
+```toml
+[[versions]] # New Line
+  version = "v1.13.0" # New Line
+  url = "https://kyverno.io" # New Line
 
-3. Create a PR.
+[[versions]]
+  version = "v1.11.0"
```
Comment (realshuting) [inline README.md:142]:
```suggestion
  version = "v1.12.0"
```

File: README.md [hunk 2]
```diff
@@ -120,17 +120,38 @@ To create a new release branch:
 
 In the `main` branch:
 
-1. Update the versions list in [params.toml](/config/_default/params.toml) to add the next release.
+1. Add a new menu version corresponding to the new release branch in [params.toml](/config/_default/params.toml) that points to https://kyverno.io below these lines:
+
+```toml
+# version_menu = "Versions"
+# Add your release versions here
+[[menu.versions]]
```
Comment (realshuting) [inline README.md:128]: I can't find this. Is it `version_menu = "v1.13.0"`?

Comment (KhaledEmaraDev) [reply README.md:128]: No, it's the section for menu entries. I copy pasted it. Should be the last thing in the file.

**Review by realshuting — APPROVED** [review]

**Comment by JimBugwadia** [comment]
@realshuting @KhaledEmaraDev - is this ready to be merged?

**Pull request merged**